res/data/btower_canm.json

```diff
@@ -0,0 +1,11 @@
+{
+    "kind": "anim",
+    "graphics": [
+        {
+            "narc": 123,
+            "file": 3,
+            "compressed": false,
+            "id": 0
+        }
```

Comment:
**suggestion**: I think that, in general, I would rather we get these to map to NARCs and NAIX files. Maybe this means that the conversion utility should be in C code so that we can easily import the necessary header files.